server/adapters/snowflake.go

```diff
@@ -176,6 +186,31 @@ func (s *Snowflake) CreateTable(tableSchema *Table) error {
 	return wrappedTx.tx.Commit()
 }
 
+// DeleteTable removes database table with name provided in Table representation
+func (s *Snowflake) DeleteTable(table *Table) error {
+	wrappedTx, err := s.OpenTx()
+	if err != nil {
+		return err
+	}
+
+	query := fmt.Sprintf(deleteSFTableTemplate, s.config.Schema, reformatValue(table.Name))
+	s.queryLogger.LogDDL(query)
+	createStmt, err := wrappedTx.tx.PrepareContext(s.ctx, query)
```

Comment:
We can use here wrappedTx.tx.ExecContext(s.ctx, query).
Without creating prepared statement

Reply:
Changed for using without a statement